spec/foundation_rails_helper/form_builder_spec.rb

```diff
@@ -58,32 +58,32 @@
     context "when input field has a prefix" do
       before do
         form_for(@author) do |builder|
-          @node = Capybara.string builder.text_field(:login, :prefix => {small: 2, medium:4, large: 6, value: "Prefix"})
+          @node = Capybara.string builder.text_field(:login, :prefix => {small: 2, medium:4, large: 6, value: "Prefix", input_style: 'radius'})
```

Comment:
Where are the tests when `input_style` is not set?  Instead of adding `input_style` here, keep the tests how they are and add some simple tests below that check the class when it _is_ set.